src/solver/implicitgemm_util.hpp

```diff
@@ -688,7 +683,7 @@ static inline bool use_amd_inline_asm(const ConvolutionContext& ctx)
     return !miopen::IsDisabled(MIOPEN_DEBUG_IMPLICIT_GEMM_NON_XDLOPS_INLINE_ASM{});
 }
 
-static inline bool support_amd_buffer_atomic_add(const ConvolutionContext& ctx)
+static inline bool support_amd_buffer_atomic_fadd(const ConvolutionContext& ctx)
 {
     const auto device_name = ctx.GetStream().GetDeviceName();
     return StartsWith(device_name, "gfx908") && ctx.IsFp32();
```

Comment:
does it must call ctx.IsFp32()? sometime data is fp16, but  it can use fp32 atomic add by using workspace

Reply:
fixed. Thanks for pointing out